app/helpers/container_helper/textual_summary.rb

```diff
@@ -52,4 +52,29 @@ def textual_ems
     end
     h
   end
+
+  def textual_container_group
+    create_link(["container_group"])
+  end
+
+  def textual_container_project
+    create_link(["container_group", "container_project"])
+  end
+
+  def create_link(reference_list)
```

Comment:
For this you could rebase on top of https://github.com/ManageIQ/manageiq/pull/3402